src/settingsKeyMapping.js

```diff
@@ -689,7 +689,49 @@ const settingsKeyMapping = {
         label: i18n.t('Metadata Versioning'),
         type: 'metadataSettings',
     },
-
+    /* ============================================================================================================ */
+    /* Category: Scheduled jobs                                                                                    */
+    /* ============================================================================================================ */
+    jobsRescheduleAfterMinutes: {
+        label: i18n.t(
+            'Number of minutes after which a job is considered stale (1-60)'
+        ),
+        type: 'textfield',
+        inputType: 'number',
+        minValue: 1,
+        maxValue: 60,
+        validators: ['positive_number'],
+    },
+    jobsCleanupAfterMinutes: {
+        label: i18n.t(
+            'Number of minutes after which a completed one-run job is deleted (1+)'
+        ),
+        type: 'textfield',
+        inputType: 'number',
+        minValue: 1,
+        maxValue: 2147483647,
+        validators: ['positive_number'],
+    },
+    jobsMaxCronDelayHours: {
+        label: i18n.t(
+            'Number of hours a job will trigger after its intended time if job has not yet run (1-24)'
+        ),
+        type: 'textfield',
+        inputType: 'number',
+        minValue: 1,
+        maxValue: 24,
+        validators: ['positive_number'],
+    },
+    jobsLogDebugBelowSeconds: {
+        label: i18n.t(
+            'Job execution interval (seconds) below which a job will be logged at debug (rather than info) level (20+)'
```

Comment:
There seems to be no message in the `en.pot` file for this one. No sure this is required but I would have expected it given the other new settings have an entry there.

Reply:
This file's auto-generated, and it will be re-generated on the next updated